demo/plugins.txt

```diff
@@ -9,16 +9,18 @@ git-server:1.6
 handlebars:1.1.1
 icon-shim:2.0.3
 jquery-detached:1.2.1
-mapdb-api:1.0.6.0
+junit:1.12
+mailer:1.17
+matrix-project:1.6
```

Comment:
Dependencies of some other plugins; nothing is bundled in 2.0 so we need to enumerate what we actually want to include.